input/badfunc03.c:4:16:Error: Redefinition of 'x'
